BowlingTest/UnitTest1.cs

```diff
@@ -0,0 +1,77 @@
+namespace BowlingTest
+{
+    public class Tests
+    {
+        // Arrange
+        Game game;
+
+        [SetUp]
+        public void Setup()
+        {
+            // Arrange
+            game = new Game();
+        }
+
+        [Test]
+        public void Hookup()
+        {
+            Assert.Pass();
+        }
+
+        [Test]
+        // Ensures balls that end up in the gutter result in a score of 0 (since no pins were hit)
+        public void GutterBalls()

```

Comment:
Test names should be more descriptive. Take a look at your comment above and see how you could turn that into a test name. Do the same with all your tests